src/common/navigator-utils.ts

```diff
@@ -31,7 +31,12 @@ export class NavigatorUtils {
         return userAgent.substring(versionOffset + versionPrefix.length + 1).split(' ')[0];
     }
 
-    public copyToClipboard(data: string): Promise<void> {
-        return this.navigatorInfo.clipboard.writeText(data);
+    public async copyToClipboard(data: string): Promise<void> {
+        try {
```

Comment:
is the try-catch needed?
can it be something like:
```typescript
public async copyToClipboard(data: string): Promise<void> {
   return this.navigatorInfo.clipboard.writeText(data).catch(error => {
      this.logger.error('Error during copyToClipboard: ', error);
      throw error;
   });
}
```

Reply:
I've changed this because I don't think it's any worse of a solution, but I also don't think it's better than the original code using try-catch; I think this is a personal style preference for which we do not currently have a consistent team style to follow, so I feel a little bit frustrated to be spending time addressing it :(